src/oops/SOLID/singleResponsibilityPrinciple/before/ApplicationLogger.java

```diff
@@ -0,0 +1,13 @@
+package oops.SOLID.singleResponsibilityPrinciple.before;
+
+public class ApplicationLogger {
+
+    public void info(String message) {
+        System.out.println("Info : " + message);
+    }
+
+    public void error(String message, Exception e) {
```

Comment:
it would be better to create a single method to print the message and re-use it in all other methods

Reply:
I think the purpose was to segregate the Logger levels for adding different methods.